### ISSUE TITLE
Does anyone reach the performance in any dataset mentioned in paper?

### ISSUE THREAD

**Issue by zk31601102** [meizhiju/layered-bilstm-crf#2]
I used default hyperparameters and pre-trained word embedding from the link the author provided.
However, we reach 60 in ACE.
Am I alone?

BTW, anyone can train the model on GPU?
I just can't load the pre-trained embedding in GPU mode, but worked on CPU mode.

**Comment by edchengg** [comment]
Hi zk,
Do you have the script to process the label? thanks
